packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -102,7 +102,7 @@ export class AuthUtils {
             const imperativeError: imperative.ImperativeError = errorDetails as imperative.ImperativeError;
             const httpErrorCode = Number(imperativeError.mDetails.errorCode);
             // open config file for missing hostname error
-            if (imperativeError.toString().includes("hostname")) {
+            if (imperativeError.toString().includes("hostname") && !imperativeError.toString().includes("protocol")) {
```

Comment:
Should we be more specific with that we are looking for (if we know the error message from Imperative in this case)?

Reply:
It's not really exported, so I am a bit concerned to do that in the event we change any of the error message. I really don't like that the logic was that if an error message contained the word hostname at all, it assumed it was blank. But I don't really know how else we would check to see if a hostname was missing on a profile or not.